corehq/apps/app_manager/suite_xml/sections/remote_requests.py

```diff
@@ -161,9 +162,18 @@ def _get_remote_request_query_datums(self):
     def _build_query_prompts(self):
         prompts = []
         for prop in self.module.search_config.properties:
+            text = Text(locale_id=id_strings.search_property_locale(self.module, prop.name))
+            if prop.hint:
+                display = Display(
+                    text=text,
+                    hint=Hint(text=Text(locale_id=id_strings.search_property_hint_locale(self.module, prop.name)))
```

Comment:
is `display.hint` just `None` if you don't included it? If so you could simplify this if / else with:

```py
Display(
    text = text, 
    hint = Hint(...) if prop.hint else None
)
```
and keep it inline below.